src/components/PopularCoursesLayout/PopularCoursesContainer/PopularCard/styledPopularCard.ts

```diff
@@ -0,0 +1,128 @@
+import styled from "styled-components";
+import { colors, hoverBtnColor, typographyColor } from "../../../../styledConstantsColors";
+import { NavLink } from "react-router-dom";
+
+export const StyledPopularCardHeader = styled.p`
+  max-width: 428px;
```

Comment:
I think you can give this width to the parent, so that the children are automatically the same width